script/gofmtcheck.sh

```
@@ -2,7 +2,7 @@
 
 # Check gofmt
 echo "==> Checking that code complies with gofmt requirements..."
-gofmt_files=$(gofmt -l $(find . -not \( -path ./.git -prune \) -not \( -path ./vendor -prune \) -name '*.go'))
+gofmt_files=$(gofmt -l $(find . -not \( -path ./.git -prune \) -not \( -path ./vendor -prune \) -not \( -path '*/vendor' -prune \) -name '*.go'))
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)